pkgs/development/python-modules/airlift/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, poetry-core, argparse, halo, pyyaml, hiyapyco, jinja2, dotmap, requests
+, kubernetes-helm, kind, docker, pythonOlder, buildPythonPackage
+, fetchFromGitHub }:
+
+buildPythonPackage rec {
```

Comment:
Is this package a library?
If not, we should use `buildPythonApplication` instead.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function

Also, this section is helpful for you.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Oh interesting, I didn't know this existed. Thanks!

Reply:
I moved the app under `applications/misc`, although I am not sure this is the best spot for it - LMK what you think